package.json

```diff
@@ -69,9 +69,9 @@
     "sonar-runner"
   ],
   "scripts": {
-    "build": "npm ci && npm run ts-build && npm run check-format && npm run license && npm test && cd tools/orchestrator",
+    "build": "npm ci && npm run ts-build && npm run check-format && npm run license && npm test && cd tools/orchestrator && npm run build",
```

Comment:
Isn't it the case that the `npm run build` at the end will recursively call the `build` script, indefinitely?

Reply:
no because it runs `npm run build` on the `tools\orchestrator` path, which has its own `package.json`

Reply:
right, sorry I read it too quickly 🤦‍♂️